src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -713,11 +713,11 @@ PeepDistance GetClosestPeep(const ScreenCoordsXY& viewportCoords, const int32_t
 {
     for (auto peep : EntityList<T>())
     {
-        if (peep->sprite_left == LOCATION_NULL)
+        if (peep->x == LOCATION_NULL)
```

Comment:
This is changed to x as both sprite_left and x are set to LOCATION_NULL under the same situations so no need to dirty both of them with the null state.